src/helpers/helpers.ts

```diff
@@ -22,7 +22,7 @@ export const customPageView = (url: string) => {
 
 export const customOutboundLink = (url: string) =>
   develop
-    ? window.open(url, "_blank")
+    ? Object.assign(document.createElement('a'), { target: '_blank', href: url, rel: 'noopener noreferrer'}).click()
     : outboundLink({ label: url }, () => {
-        window.open(url, "_blank");
-      });
+      Object.assign(document.createElement('a'), { target: '_blank', href: 'http://google.com', rel: 'noopener noreferrer'}).click()
```

Comment:
the href here redirects only to google.com shouldn't it be the url param?

Reply:
Ohhh sorry my mistake :)

Reply:
Changes done